app/views/qa/_state_button.html.erb

```diff
@@ -7,21 +7,27 @@
     <span class="visually-hidden">Toggle Dropdown</span>
   </button>
   <div class="dropdown-menu">
-    <%= f.collection_radio_buttons(:state, record.class.states, :first, :first) do |b| %>
-      <%= b.label class: 'state' do %>
-        <%= b.radio_button class: 'd-none', data: { behavior: 'state-radio'} %>
+    <%= f.collection_radio_buttons(:state, states, :first, :first) do |b| %>
+      <% label_options = {} %>
```

Comment:
Can we clean this up at all? Maybe in a helper? 
We can pass in the state to a method that returns an empty hash or
```ruby
{ data: { bs_toggle: 'tooltip', bs_title: 'You are not a Reviewer for this project.' } }
```
and call it inline on L15

Or in general can we structure this so that we have an object that provides us with what we'll need ex.
label_options, disabled: true/false, state-label and we just pull each section in the view?
```ruby
{ published: { disabled: true, label_options: {}, state_label: 'Content is final, ready for the report.' }
```